### PR TITLE
DM-25518: update for compatibility with ts_salobj 5-6

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py:3]: Telescope and Site

File: python/lsst/ts/externalscripts/auxtel/calsys_takedata.py [hunk 2]
```diff
@@ -1,3 +1,23 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline python/lsst/ts/externalscripts/auxtel/calsys_takedata.py:3]: Telescope and Site

File: python/lsst/ts/externalscripts/coordination/laser_coordination.py [hunk 3]
```diff
@@ -1,3 +1,23 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline python/lsst/ts/externalscripts/coordination/laser_coordination.py:3]: Telescope and Site

File: scripts/auxtel/at_cam_take_image.py [hunk 4]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/at_cam_take_image.py:4]: Telescope and Site

File: scripts/auxtel/calsys_take_narrowband_data.py [hunk 5]
```diff
@@ -1,5 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/calsys_take_narrowband_data.py:4]: Telescope and Site

File: scripts/auxtel/calsys_takedata.py [hunk 6]
```diff
@@ -1,5 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/calsys_takedata.py:4]: Telescope and Site

File: scripts/auxtel/latiss_acquire_and_take_sequence.py [hunk 7]
```diff
@@ -1,6 +1,26 @@
 #!/usr/bin/env python
-from lsst.ts.externalscripts.auxtel.latiss_acquire_and_take_sequence import (
-    LatissAcquireAndTakeSequence,
-)
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/latiss_acquire_and_take_sequence.py:4]: Telescope and Site

File: scripts/auxtel/latiss_acquire_target.py [hunk 8]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
-from lsst.ts.externalscripts.auxtel.latiss_acquire_target import LatissAcquireTarget
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/latiss_acquire_target.py:4]: Telescope and Site

File: scripts/auxtel/latiss_take_sequence.py [hunk 9]
```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env python
-from lsst.ts.externalscripts.auxtel.latiss_acquire_target import LatissTakeSequence
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/auxtel/latiss_take_sequence.py:4]: Telescope and Site

File: scripts/main/laser_coordination.py [hunk 10]
```diff
@@ -1,6 +1,26 @@
 #!/usr/bin/env python
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline scripts/main/laser_coordination.py:4]: Telescope and Site

File: tests/auxtel/test_calsys_take_narrowband_data.py [hunk 11]
```diff
@@ -1,3 +1,23 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline tests/auxtel/test_calsys_take_narrowband_data.py:3]: Telescope and Site

File: tests/auxtel/test_calsys_takedata.py [hunk 12]
```diff
@@ -1,3 +1,23 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline tests/auxtel/test_calsys_takedata.py:3]: Telescope and Site

**Comment by r-owen** [comment]
Fixed with a global search and replace.

**Pull request merged**